docs/Playing/index.rst

```diff
@@ -6,6 +6,7 @@ Freeciv21 and any of the varying rulesets that the Longturn.net community likes
 
 
 .. toctree::
+  LT-Guide/index.rst
```

Comment:
I consider the LT Guide and the How To Play to be closely related. Therefore I would not separate the both guides withe FAQ entry. Additionally I would propose to let the general Guide come first, followed by the more specific guide.

1. How To Play
2. LT Guide
3. FAQ
4. ...

Reply:
Another good call.